### PR TITLE
Synch repo scc manifest

### PR DESCRIPTION
**Pull request by maiqueb**
**What this PR does / why we need it**:
This PR synchronizes the `macvtap-cni` repo manifests with the defined templates.

It furthermore now adds a check to CI, ensuring these are kept synchronized.

**Special notes for your reviewer**:
Depends-on: #66 

**Release note**:

```release-note
Specify a valid SCC in the manifests.
```

### CODE REVIEW

File: automation/check-patch.setup.sh [hunk 1]
```diff
@@ -27,3 +27,13 @@ cp -rf $(pwd)/. ${TMP_PROJECT_PATH}
 
 echo 'Exporting temporary project path'
 export TMP_PROJECT_PATH
+
+echo 'Ensuring the manifests are in synch'
```
Comment (phoracek) [inline automation/check-patch.setup.sh:31]:
```suggestion
echo 'Ensuring the manifests are in sync'
```

File: automation/check-patch.setup.sh [hunk 2]
```diff
@@ -27,3 +27,13 @@ cp -rf $(pwd)/. ${TMP_PROJECT_PATH}
...
+if [[ -n $(git status --porcelain 2>/dev/null) ]]; then
+    echo "ERROR: git tree state is not clean!"
+    echo "Run `make manifests` and commit those changes"
+    git status
+    git diff
+    exit 1
+fi
+
```
Comment (phoracek) [inline automation/check-patch.setup.sh:39]:
```suggestion
```

File: automation/check-patch.setup.sh (hunk 1 above)
Comment (maiqueb) [reply automation/check-patch.setup.sh]: Done.

**Review by phoracek — COMMENTED** [review]
/lgtm

**Comment by maiqueb** [comment]
/hold cancel 

#66 was merged.

**Pull request merged**